src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
 public class FileWork {
+    private static final String STRING_START_WITH = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        File file = new File(fileName);
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(file))) {
+            StringBuilder stringBuilder = new StringBuilder();
+            String lineSeparator = System.getProperty("line.separator");
+            String lineFromFile = bufferedReader.readLine();
+            while (lineFromFile != null) {
+                stringBuilder.append(lineFromFile).append(lineSeparator);
+                lineFromFile = bufferedReader.readLine();
+            }
+            String[] splitedStringBuilderResult = stringBuilder
+                    .toString()
+                    .toLowerCase()
+                    .split(" |\\.|,|:|;|'|!|\\?|\r|\n");
```

Comment:
let's maybe replace all punctuation chars with `""`, and then split by `"\\s+"`
take a look what does this regex mean

Reply:
Thanks! Done